src/dashboard/src/components/administration/views.py

```diff
@@ -223,6 +227,127 @@ def storage(request):
     system_directory_description = 'Available storage'
     return render(request, 'administration/locations.html', locals())
 
+def usage(request):
+    usage_dirs = _usage_dirs()
+
+    return render(request, 'administration/usage.html', locals())
+
+def _usage_dirs(calculate_usage = True):
+    # Get shared directory path
+    clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
+    config = ConfigParser.SafeConfigParser()
+    config.read(clientConfigFilePath)
+
+    shared_dir = config.get('MCPClient', 'sharedDirectoryMounted')
+
+    # Get DIP uploads directory
+    dip_upload_dir = os.path.join(shared_dir, 'watchedDirectories/uploadDIP')
+
+    if calculate_usage:
+        # Get shared directory overall size and use
+        shared_size = _usage_check_directory_volume_size(shared_dir)
+        shared_used = _usage_check_directory_used(shared_dir)
+
+        # Get DIP upload usage
+        dip_upload_used = _usage_check_directory_used(dip_upload_dir)
+    else:
+        shared_size = None
+        shared_used = None
+        dip_upload_used = None
+
+    return {
+        'shared': {
+            'path': shared_dir,
+            'size': shared_size,
+            'used': shared_used
+        },
+        'dips': {
+            'description': 'DIP uploads',
+            'path': dip_upload_dir,
+            'size': shared_size,
+            'used': dip_upload_used
+        },
+        'backlog': _usage_location_info('BL', shared_size)
```

Comment:
It would be nice if this was easier to add more directories to this function.  Maybe have a dict (or an OrderedDict?) with the static info for all the directories you want to display, and iterate over them and populate the size/used if calculate_usage=True?